lib/executor/ramping_vus.go

```diff
@@ -273,18 +277,19 @@ func (vlvc RampingVUsConfig) precalculateTheRequiredSteps(et *lib.ExecutionTuple
 // iterations), where stars represent actively scheduled VUs and dots are used
 // for VUs that are potentially finishing up iterations:
 //
+//	^
+//	|
 //
-//      ^
-//      |
 // VUs 6|  *..............................
-//     5| ***.......*..............................
-//     4|*****.....***.....**..............................
-//     3|******...*****...***..............................
-//     2|*******.*******.****..............................
-//     1|***********************..............................
-//     0--------------------------------------------------------> time(s)
-//       012345678901234567890123456789012345678901234567890123   (t%10)
-//       000000000011111111112222222222333333333344444444445555   (t/10)
+//
+//	5| ***.......*..............................
+//	4|*****.....***.....**..............................
+//	3|******...*****...***..............................
+//	2|*******.*******.****..............................
+//	1|***********************..............................
+//	0--------------------------------------------------------> time(s)
+//	  012345678901234567890123456789012345678901234567890123   (t%10)
+//	  000000000011111111112222222222333333333344444444445555   (t/10)
```

Comment:
This is even worse then the one above :(

Reply:
ah, I missed that :man_facepalming: will fix it

Reply:
Resolved by the [latest force-push](https://github.com/grafana/k6/compare/fcdcb2237c5a91a51c406d6bb80d27db592c4ec6..07630ed366716acf6b5e0675946301188d3b23a9)